tests/unit_tests/test_temp_interp.py

```diff
@@ -86,6 +86,70 @@ def cross_section(value):
     lib.export_to_xml('cross_sections_fake.xml')
 
 
+def fake_thermal_scattering():
+    """Create a fake thermal scattering library for U-235 at 294K and 600K
+    """
+    fake_tsl = openmc.data.ThermalScattering("c_U_fake", 1.9968, 4.9, [0.0253])
+    fake_tsl.nuclides = ['U235']
+
+    # Create elastic reaction
+    bragg_edges = [0.00370672, 0.00494229]
+    factors = [0.00375735, 0.01386287]
+    coherent_xs = openmc.data.CoherentElastic(bragg_edges, factors)
+    incoherent_xs_294 = openmc.data.Tabulated1D([0.00370672, 0.00370672], [0.00370672, 0.00370672])
+    elastic_xs_base = openmc.data.Sum((coherent_xs, incoherent_xs_294))
+    elastic_xs = {'294K': elastic_xs_base, '600K': elastic_xs_base}
+    coherent_dist = openmc.data.CoherentElasticAE(coherent_xs)
+    incoherent_dist_294 = openmc.data.IncoherentElasticAEDiscrete([
+        [-0.6, -0.18, 0.18, 0.6], [-0.6, -0.18, 0.18, 0.6]
+    ])
+    incoherent_dist_600 = openmc.data.IncoherentElasticAEDiscrete([
+        [-0.1, -0.2, 0.2, 0.1], [-0.1, -0.2, 0.2, 0.1]
+    ])
+    elastic_dist = {
+        '294K': openmc.data.MixedElasticAE(coherent_dist, incoherent_dist_294),
+        '600K': openmc.data.MixedElasticAE(coherent_dist, incoherent_dist_600)
+        }
+    fake_tsl.elastic = openmc.data.ThermalScatteringReaction(elastic_xs, elastic_dist)
+
+    # Create inelastic reaction
+    inelastic_xs = {
+        '294K': openmc.data.Tabulated1D([1.0e-5, 4.9], [13.4, 3.35]),
+        '600K': openmc.data.Tabulated1D([1.0e-2, 10], [1.4, 5])
+        }
+    breakpoints = [3]
+    interpolation = [2]
+    energy = [1.0e-5, 4.3e-2, 4.9]
+    energy_out = [
+        openmc.data.Tabular([0.0002, 0.067, 0.146, 0.366], [0.25, 0.25, 0.25, 0.25]),
+        openmc.data.Tabular([0.0001, 0.009, 0.137, 0.277], [0.25, 0.25, 0.25, 0.25]),
+        openmc.data.Tabular([0.0579, 4.555, 4.803, 4.874], [0.25, 0.25, 0.25, 0.25]),
+    ]
+    for eout in energy_out:
+        eout.normalize()
+        eout.c = eout.cdf()
+    discrete = openmc.stats.Discrete([-0.9, -0.6, -0.3, -0.1, 0.1, 0.3, 0.6, 0.9], [1/8]*8)
+    discrete.c = discrete.cdf()[1:]
+    mu = [[discrete]*4]*3
+    dist = openmc.data.IncoherentInelasticAE(
+        breakpoints, interpolation, energy, energy_out, mu)
+    inelastic_dist = {'294K': dist, '600K': dist}
+    inelastic = openmc.data.ThermalScatteringReaction(inelastic_xs, inelastic_dist)
+    fake_tsl.inelastic = inelastic
+
+    return fake_tsl
+
+
+def edit_fake_cross_sections():
+    """Edit the test cross sections xml to include fake thermal scattering data
+    """
+    lib = openmc.data.DataLibrary.from_xml("cross_sections_fake.xml")
+    c_U_fake = fake_thermal_scattering()
+    c_U_fake.export_to_hdf5("c_U_fake.h5")
+    lib.register_file("c_U_fake.h5")
+    lib.export_to_xml("cross_sections_fake.xml")
```

Comment:
Rather than having this function modify the fake cross sections that were already set up, I think we should just create both the fake U235 and thermal scattering data together (in `make_fake_cross_section`) so that there's no need to edit it after the fact. Does that make sense?

Reply:
Yep, that makes total sense! I originally didn't want to mess with the existing tests, but this shouldn't have an impact.